### ISSUE TITLE
java.lang.NoClassDefFoundError (FirebaseInstanceId) with latest flutter update version 2.2.1 stable channel

### ISSUE THREAD

**Issue by SaravanarajaGITHUB** [hypertrack/sdk-flutter#10]
![HyperTrack_error](https://user-images.githubusercontent.com/6235704/120217861-8fc59a00-c256-11eb-9ba0-206e75a413b4.png)

With flutter update 2.2.1 facing this issue.

hypertrack_plugin: ^0.1.7 is the version I used before after this issue tried with ^0.1.9 but still facing the same issue.

**UPDATED more info:** below is the sample project screenshot where u can see my simple dependency used in the project.

![image](https://user-images.githubusercontent.com/6235704/120975651-19adbf80-c78f-11eb-8033-e32bca9350bb.png)

when I didn't use firebase-related dependency, my HyperTrack plugin initialization works fine, but with firebase dependency, I'm getting the error.

**Comment by SaravanarajaGITHUB** [comment]
I see a similar error is araising in this thread, which is live on 6 days ago. 

link --> http://5.9.10.113/67686867/onesignal-android-studio-unknown-error-getting-fcm-token

**Comment by bikcrum** [comment]
Add this your `app/build.gradle` 

```
dependencies {
   ...
    implementation "com.google.firebase:firebase-iid"
}
```

**Comment by deniszpua** [comment]
@SaravanarajaGITHUB This is fixed in [v0.2.0](https://pub.dev/packages/hypertrack_plugin/versions/0.2.0)
